engine/worker/cmd_artifacts.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/sdk"
+)
+
+func cmdArtifacts(w *currentWorker) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "artifacts",
+		Short: "worker artifacts [--workflow=<workflow-name>] [--number=<run-number>] [--tag=<tag>] [--pattern=<pattern>]",
+		Long: `
+Inside a job, you can list artifacts of a workflow:
+
+	worker artifacts --pattern="files.*.yml"
+
+	#theses two commands have the same result:
+	worker artifacts
+	worker artifacts --workflow={{.cds.workflow}} --number={{.cds.run.number}}
```

Comment:
don't understand why we need the second line

Reply:
coherence with worker download